src/analyze/analyze.c

```diff
@@ -1972,7 +1981,7 @@ static int dump_filesystems(int argc, char *argv[], void *userdata) {
                 if (k < 0) {
                         fputc('\n', stdout);
                         fflush(stdout);
-                        log_notice_errno(k, "# Not showing unlisted filesystems, couldn't retrieve kernel filesystem list: %m");
+                                log_notice_errno(k, "# Not showing unlisted filesystems, couldn't retrieve kernel filesystem list: %m");
```

Comment:
what's with this indentation change?

Reply:
Initially I wanted to add `if (!arg_quiet)` here, but then I added an early exit above. Thanks for catching this.